Signal/src/ViewControllers/NewContactThreadViewController.m

```diff
@@ -295,6 +283,30 @@ - (void)updateTableContents
 
     __weak NewContactThreadViewController *weakSelf = self;
 
+    if (self.contactsViewHelper.contactsManager.isSystemContactsDenied) {
```

Comment:
How do we update this if the user leaves app to grant/deny access?  Should we reload the table on "become active"?

Reply:
Changing contact permissions kills the app.

Reply:
Can you add a comment to this effect please?

Reply:
sure, done.